syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -120,17 +145,24 @@ class Syft internal constructor(
             )
         } ?: executeAuthentication(job)
     }
-
+    
+    // method to get the state of the Syft Worker.
     override fun isDisposed() = isDisposed.get()
 
+   // method to dispose of a Syft Worker.
```

Comment:
put this in Kdoc rather than inline comment.
- Once a worker is disposed off, all the jobs that are running are disposed as well.